### ISSUE TITLE
Add gitlab to social

### ISSUE THREAD

**Issue by ltbinsbe** [jekyll/minima#547]
Right underneath/above github

**Comment by MaxValue** [comment]
This is already supported, see https://github.com/jekyll/minima#social-networks
It can be that you are using version 2.5.1 which does not support social links yet.

**Comment by p2635** [comment]
How do we update to minima 3 when it's not even in releases yet?

**Comment by ashmaroli** [comment]
@p2635 By using the `jekyll-remote-theme` plugin.
If you're deploying via GitHub Pages, simply list the plugin under `plugins:` key in your config file and replace `theme:` key with `remote_theme:`:
```diff
  # plugins needed by the site
  plugins:
  - jekyll-feed
  - jekyll-seo-tag
+ - jekyll-remote-theme

  # the theme used by the site
- theme: minima
+ remote_theme: jekyll/minima
```

**Comment by p2635** [comment]
Ah ok thanks @ashmaroli , I'm on Gitlab actually so does that change the advice you're giving?

**Comment by ashmaroli** [comment]
Yes. I have no idea if `jekyll-remote-theme` works on Gitlab :stuck_out_tongue: 
So, you will have to instead ditch the config file changes and use Bundler to load the theme from the repository via the Gemfile:
```diff
  # Gemfile

- gem "minima", "~> 2.0"
+ gem "minima", github: "jekyll/minima"
```

**Comment by p2635** [comment]
No luck I'm afraid @ashmaroli 

Did a bundle update and it says 'Using minima 2.5.1 from https://github.com/jekyll/minima.git (at master@3cdd14d)'

**Comment by ashmaroli** [comment]
@p2635 Don't be misled by the version number. The `master` branch is still `v2.5.1`. It'll only be `v3.0.0` during release.
So, you're good to go.

**Comment by p2635** [comment]
I wish that was the case @ashmaroli .. I still don't see the social links coming up.

Output for `bundle info minima`, would you expect to see v2.5.1 here as well?
```
  * minima (2.5.1 3cdd14d)
        Summary: A beautiful, minimal theme for Jekyll.
        Homepage: https://github.com/jekyll/minima
        Path: /home/phil/.gem/ruby/2.7.0/bundler/gems/minima-3cdd14dff121
```

Quickly added this to my yml file to test:

```
# Social
minima:
  social_links:
    twitter: poop
```

Any other ideas? Thanks for helping out.

**Comment by ashmaroli** [comment]
@p2635 Only way to actually debug would be if I had a link to your public repository.

**Comment by MaxValue** [comment]
Other ways:

- copy the desired theme version directly into your project. I never tried this.
- build your site locally (using the jekyll-remote-theme plugin) and then upload the built site (_site) to your gitlab pages repo. you will have to change your ci script so it serves a static site, because now the files are already rendered and do not need to be rendered by gitlab.

**Comment by p2635** [comment]
> @p2635 Don't be misled by the version number. The master branch is still v2.5.1. It'll only be v3.0.0 during release.
So, you're good to go.

What I don't understand is whether 2.5.1 in the project releases here = minima 2.5.1 or minima v3.0.0. Surely the fix here is just to release 3.0.0? If v3 was released here, I won't have these issues of trying to specifically copying and doing these workarounds suggested by @MaxValue . 

Am I missing something here?

**Comment by MaxValue** [comment]
the master (default) branch here is minima 3.X.X There is no actual version number, because that version hasn't been released yet.

So, when you install minima via the Gemfile (and bundler) it downloads the gem from rubygems which has the 2.5.1 version registered as the latest version.

When you use the jekyll-remote-theme plugin, the plugin downloads the theme directly from this repository again each time you build your site. I guess it looks at the default branch which is master which contains the 3.X.X version.

I am not a maintainer, but I am wondering too why this version isn't "released" yet. There is issue #411 which already requests a new release, but no reason why this is not carried out. Btw, the thread in that issue also contains some tips on how to include a particular version of the theme.
